src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -776,12 +786,12 @@ pub fn new_sub_level(
 
 pub fn add_ssts_to_sub_level(
     l0: &mut OverlappingLevel,
-    sub_level_idx: usize,
+    sub_level_insert_hint: Result<usize, usize>,
```

Comment:
> We don't have to pass such hint here. For `add_ssts_to_sub_level` it can always mean adding sst to an existing level, so we can still only pass the `sub_level_idx`, and then we don't have to call `insert_new_sub_level` here. And the parameter `level_type` and `insert_table_infos` can be removed.
> 
> `insert_new_sub_level` can be called in `init_with_parent_group` when `insert_hint` matches `Err(idx)`.

ok.

Reply:
> We don't have to pass such hint here. For `add_ssts_to_sub_level` it can always mean adding sst to an existing level, so we can still only pass the `sub_level_idx`, and then we don't have to call `insert_new_sub_level` here. And the parameter `level_type` and `insert_table_infos` can be removed.
> 
> `insert_new_sub_level` can be called in `init_with_parent_group` when `insert_hint` matches `Err(idx)`.

ok.